frontend/src/pages/ConnectedPages/Game/index.tsx

```diff
@@ -17,12 +17,24 @@ const Game: React.FC<RouteComponentProps> = ({ location }) => {
   const [previewUrl, setPreviewUrl] = useState<string>("");
   const [answers, setAnswers] = useState<Answer[]>([]);
   const [score] = useState<number>(0);
+  const player = useRef<null | HTMLAudioElement>(null);
 
-  const onQuestionUpdateReceived = (question: QuestionUpdateMessage): void => {
-    console.log(`Received question ${question}`);
-    setPreviewUrl(question.previewUrl);
-    setAnswers(question.answers);
+  useEffect(() => {
+    (async function reloadPlayer() {
```

Comment:
This async await is not necesarry. Just delete it.

Reply:
Actually it is, because `play` is an async function. Without the `async await`, the console is displaying an error every time the song is switched.

Reply:
You are completely rigth, I just read the demo. Can you at least put the function on the bottom of the useEffect, to separate the 2 like it is done in ```ConnectedPages/index``` ?